website/docs/plugin/which-sdk.mdx

```diff
@@ -10,25 +10,16 @@ description: >-
 There are two SDKs that you can use to build Terraform providers:
 
 * [SDKv2](/plugin/sdkv2) is what the majority of existing providers are built on, and it continues to provide a stable development experience.
-* [Terraform Plugin Framework](/plugin/framework) is a new SDK under active development, realigning the provider development experience and abstractions around the updated architecture of Terraform. It represents our vision of the future of Terraform plugin development.
+* [Terraform Plugin Framework](/plugin/framework) is a new SDK, realigning the provider development experience and abstractions around the updated architecture of Terraform. It represents our vision of the future of Terraform plugin development.
 
 This guide will help you decide whether you should continue using SDKv2, or begin using the framework.
```

Comment:
```suggestion
This guide helps you decide whether you should continue using SDKv2 or begin using the framework.
```

Suggest editing to avoid a future tense style issue